nconnect.go

```diff
@@ -514,13 +541,14 @@ func (nc *nconnect) startSSAndTunnel() {
 		os.Exit(0)
 	}()
 
-	go func() {
-		err := nc.tunnel.Start()
-		if err != nil {
-			log.Fatal(err)
-		}
-		os.Exit(0)
-	}()
+	for _, t := range nc.tunnels {
+		go func(t *tunnel.Tunnel) {
+			err := t.Start()
+			if err != nil {
+				log.Printf("tunnel %v start err %v", t.Addr(), err)
```

Comment:
I think we should call os.exit like before when any tunnel exits

Reply:
Yes, os.exit is added.

Reply:
The logic is different from the old one, please double check

Reply:
Yes, modified.